runner.mjs

```diff
@@ -9,9 +9,11 @@ export default function run (files, report = () => {}) {
   let passed = 0
   let failed = 0
   let crashed = 0
+  let pending = 0
```

Comment:
This keeps track of the *total number of pending tests for all processes combined*